CollAction/Views/Projects/Details.cshtml

```diff
@@ -27,10 +27,20 @@
     <div class="container">
         @{
             string disableCommit = "";
+            string commitText = "Take Part";
             if (!Model.Project.IsActive)
             {
                 disableCommit = "disable-commit";
             }
+            if (Model.ExternalStatus == ProjectExternalStatus.ComingSoon)
```

Comment:
Maybe commit text should be a property of the viewmodel?

Reply:
I left this one as is. The text is pretty specific to the view and the ExternalStatus might get used for something else in the future.